bin/Powheg/examples/ggHH_quark-mass-effects_withJHUGen_PDF4LHC_13TeV/ggHH.powheg.input

```diff
@@ -0,0 +1,104 @@
+! ggHH production parameters:
+mtdep 2              ! 0: Higgs effective field theory (HEFT)
+!                      1: Born improved HEFT
+!                      2: approximated full theory (FTapprox)
+!                      3: full theory
+
+hmass 125            ! Higgs boson mass
+topmass 173          ! top quark mass
+hdecaymode -1        ! PDG code for Higgs boson decay products (it affects only the SMC)
+!                      allowed values are:  
+!                      0 all decay channels open
+!                      1-6 d dbar, u ubar,..., t tbar (as in HERWIG)
+!                      7-9 e+ e-, mu+ mu-, tau+ tau-
+!                      10  W+W-
+!                      11  ZZ
+!                      12  gamma gamma      
+!                      -1  all decay channels closed
+
+! general parameters:
+numevts  NEVENTS     ! number of events to be generated
+ih1         1        ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2         1        ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 7000d0        ! energy of beam 1
+ebeam2 7000d0        ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1 306000        ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000	     ! pdf set for hadron 2 (LHA numbering)
+
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1    ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1    ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+!
+ncall1  10000        ! number of calls for initializing the integration grid
+itmx1       3        ! number of iterations for initializing the integration grid
+ncall2  10000        ! number of calls for computing the integral and finding upper bound
+itmx2       3        ! number of iterations for computing the integral and finding upper bound
+foldcsi     1        ! number of folds on csi integration
+foldy       1        ! number of folds on  y  integration
+foldphi     1        ! number of folds on phi integration
+nubound  5000        ! number of bbarra calls to setup norm of upper bounding function
+# icsimax   1        ! <= 100, number of csi subdivision when computing the upper bounds
+# iymax     1        ! <= 100, number of y subdivision when computing the upper bounds
+# xupbound 2d0       ! increase upper bound for radiation generation
```

Comment:
did you play with the parameters for a smooth integration grid? These parameters seem to be the default? 
(see [this twiki](https://twiki.cern.ch/twiki/bin/viewauth/CMS/PowhegBOXPrecompiledCheckGrids))

Reply:
usually the parameters in the example datacards in POWHEG are those used in the paper, which need to be multiplied by the number of cores...
indeed they look pretty small to me

Reply:
@agrohsje , do you have an answer?

Reply:
have you tried running and checking the integration grids as descripted in 
https://twiki.cern.ch/twiki/bin/viewauth/CMS/PowhegBOXPrecompiledCheckGrids

Reply:
Yes I did, it looks pretty smooth https://www.dropbox.com/s/w1kdnxxyp7nm82q/powegGridHH.PNG?dl=0